### PR TITLE
Add ckan29 integration

### PR DESCRIPTION
**Pull request by kentsanggds**
## What

Deploy CKAN 2.9 settings and necessary files onto Integration and startup worker processes.

The intention is to leave CKAN 2.9 running on Integration for a week to see if there are any issues running harvest jobs or with the GUI or managing users / datasets / harvest sources.

## Reference 

https://trello.com/c/GZP41t9N/2553-deploy-ckan-29-onto-integration

### CODE REVIEW

File: hieradata_aws/common.yaml [hunk 1]
```diff
@@ -64,6 +64,9 @@ node_class: &node_class
   ckan:
     apps:
       - ckan
+  ckan29:
```
Comment (fredericfran-gds) [inline hieradata_aws/common.yaml:67]: I don't get this, this Hiera change is instructing that there will be a new class of machine `ckan29` and install `ckan`.

Comment (fredericfran-gds) [reply hieradata_aws/common.yaml:67]: Haven't read through the code, I don't think **any** change to **this** file is needed.

**Review by fredericfran-gds — COMMENTED** [review]
I have made one comment & I guess you have done some extensive testing in integration so happy to approve if you fix my comment.

**Comment by fredericfran-gds** [comment]
Another comment is if you could squash the 20 commits into a smaller number of commits, maybe 1?

File: hieradata_aws/common.yaml (hunk 1 above)
Comment (kentsanggds) [reply hieradata_aws/common.yaml:67]: There is some logic in the `ckan` app which deploys files according to the environment so no need to create a `ckan29` app. I did try this route initially but hit a number of issues.

**Comment by kentsanggds** [comment]
OK, I'll squash all the commits into 1, I left it separated to make it easier to review

File: hieradata_aws/common.yaml (hunk 1 above)
Comment (fredericfran-gds) [reply hieradata_aws/common.yaml:67]: i don't see the need for this code. can you point to the logic/code which uses this hiera?

**Review by fredericfran-gds — APPROVED** [review]
looks good, this was extensively tested in integration.

**Pull request merged**